kvdb-rocksdb/src/lib.rs

```diff
@@ -443,6 +443,18 @@ impl Database {
 							stats_total_bytes += key.len();
 							batch.delete_cf(cf, &key).map_err(other_io_err)?
 						}
+						DBOp::DeletePrefix { col: _, prefix } => {
+							if prefix.len() > 0 {
+								let end_range = kvdb::end_prefix(&prefix[..]);
+								batch.delete_range_cf(cf, &prefix[..], &end_range[..]).map_err(other_io_err)?;
+							} else {
+								// delete the whole column
+								let end_range = &[255u8];
+								batch.delete_range_cf(cf, &prefix[..], &end_range[..]).map_err(other_io_err)?;
```

Comment:
why do we need to delete a range, if we're deleting the whole column later?

Reply:
I did not delete the whole column (at first I tried to but it requires mutable access and some changes overall). So I end up thinking that if we use the delete range to delete the column we are doing something wrong, so using delete range for it felt more consistent.
(I will change the comment)

Reply:
```suggestion
								batch.delete_range_cf(cf, &[], &end_range[..]).map_err(other_io_err)?;
```